exp/topology/desiredstate/desired_state_test.go

```diff
@@ -470,6 +493,33 @@ func TestComputeControlPlane(t *testing.T) {
 		assertNestedFieldUnset(g, obj, contract.ControlPlane().Replicas().Path()...)
 		assertNestedFieldUnset(g, obj, contract.ControlPlane().MachineTemplate().InfrastructureRef().Path()...)
 	})
+	t.Run("Skips setting readinessGates if required", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Skips setting readinessGates if not set in Cluster and ClusterClass", func(t *testing.T) {
```
required sounds a bit misleading